js/genkit/src/genkit.ts

```diff
@@ -247,6 +251,18 @@ export class Genkit {
     return defineSchema(this.registry, name, schema);
   }
 
+  /**
+   * Defines and registers a custom model output formatter.
```

Comment:
Can you add a little bit more here explaining how it's used like some of the other define methods do?